heatpump.yaml

```diff
@@ -2073,52 +2091,157 @@ switch:
   # Register: 0 -> Bit 0
   # When Room Temperature Control is enabled, the temperature sensor in the remote controller
   # will be used to define when the heatpump should be turned (powered) on or off
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Room Temperature Control"
+  - platform: template
+    name: "Room Temperature Control"                  #"Power Climate Control -uses FanCoilUnit as emission type"
```

Comment:
Can you remove the comment from this line

Reply:
Yes i do in a moment -i left it for suggestion for name to be more logical